sync/device.ts

```diff
@@ -0,0 +1,177 @@
+import * as Docker from 'dockerode';
+import { Dockerfile } from 'livepush';
+import * as _ from 'lodash';
+import { Builder } from 'resin-docker-build';
+
+import { child_process, fs } from 'mz';
```

Comment:
You can do
```suggestion
import { child_process } from 'mz';
import { promises as fs } from 'fs';
```
now that node 10 is the minimum